[2024-07-24T05:36:59.973+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-24T05:37:00.202+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:50:00+00:00 [queued]>
[2024-07-24T05:37:00.264+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:50:00+00:00 [queued]>
[2024-07-24T05:37:00.264+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-24T05:37:00.325+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): database_connection> on 2024-07-24 00:50:00+00:00
[2024-07-24T05:37:00.391+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=68) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-24T05:37:00.407+0000] {standard_task_runner.py:64} INFO - Started process 92 to run task
[2024-07-24T05:37:00.424+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_Fraud', 'database_connection', 'scheduled__2024-07-24T00:50:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/server_DAG.py', '--cfg-path', '/tmp/tmpx12p92rr']
[2024-07-24T05:37:00.470+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask database_connection
[2024-07-24T05:37:01.467+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:50:00+00:00 [running]> on host fc934038a502
[2024-07-24T05:37:02.250+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@test.com' AIRFLOW_CTX_DAG_OWNER='fraud_detector' AIRFLOW_CTX_DAG_ID='ETL_Fraud' AIRFLOW_CTX_TASK_ID='database_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-07-24T00:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-24T00:50:00+00:00'
[2024-07-24T05:37:02.253+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-24T05:37:02.254+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-24T05:37:02.255+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "database_connection"']
[2024-07-24T05:37:02.358+0000] {subprocess.py:86} INFO - Output:
[2024-07-24T05:37:02.375+0000] {subprocess.py:93} INFO - database_connection
[2024-07-24T05:37:02.377+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-24T05:37:02.382+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-24T05:37:02.556+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_Fraud, task_id=database_connection, run_id=scheduled__2024-07-24T00:50:00+00:00, execution_date=20240724T005000, start_date=20240724T053700, end_date=20240724T053702
[2024-07-24T05:37:02.698+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-24T05:37:02.784+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-24T05:37:02.799+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-24T09:51:54.074+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-24T09:51:54.307+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:50:00+00:00 [queued]>
[2024-07-24T09:51:54.330+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:50:00+00:00 [queued]>
[2024-07-24T09:51:54.333+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-24T09:51:54.383+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): database_connection> on 2024-07-24 00:50:00+00:00
[2024-07-24T09:51:54.421+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=78) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-24T09:51:54.425+0000] {standard_task_runner.py:64} INFO - Started process 124 to run task
[2024-07-24T09:51:54.428+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_Fraud', 'database_connection', 'scheduled__2024-07-24T00:50:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/server_DAG.py', '--cfg-path', '/tmp/tmpe6lgn1m8']
[2024-07-24T09:51:54.444+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask database_connection
[2024-07-24T09:51:54.631+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:50:00+00:00 [running]> on host 0542575fe516
[2024-07-24T09:51:55.064+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@test.com' AIRFLOW_CTX_DAG_OWNER='fraud_detector' AIRFLOW_CTX_DAG_ID='ETL_Fraud' AIRFLOW_CTX_TASK_ID='database_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-07-24T00:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-24T00:50:00+00:00'
[2024-07-24T09:51:55.067+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-24T09:51:55.069+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-24T09:51:55.071+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "database_connection"']
[2024-07-24T09:51:55.201+0000] {subprocess.py:86} INFO - Output:
[2024-07-24T09:51:55.211+0000] {subprocess.py:93} INFO - database_connection
[2024-07-24T09:51:55.214+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-24T09:51:55.215+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-24T09:51:55.406+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_Fraud, task_id=database_connection, run_id=scheduled__2024-07-24T00:50:00+00:00, execution_date=20240724T005000, start_date=20240724T095154, end_date=20240724T095155
[2024-07-24T09:51:55.574+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-24T09:51:55.711+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-24T09:51:55.718+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
